spec/javascripts/angular/filters/filters_spec.js

```diff
@@ -31,12 +51,13 @@ describe('filters', function() {
     });
 
     it('exact match', function() {
-      expect(regex('important-data.generic-data', '\.generic-data', '')).toEqual('important-data');
+      expect(regex('important-data.generic-data', '.generic-data', '')).toEqual('important-data');
```

Comment:
Why are these changes part of this commit? Doesn't seem to be related to hostname changes.